### PR TITLE
Make a few GitHub page updates

### PR DESCRIPTION
**Pull request by zachmargolis**

### CODE REVIEW

**Review by aduth — APPROVED** [review]
LGTM 👍

File: _articles/github.md [hunk 1]
```diff
@@ -142,24 +142,31 @@ To request access to GitHub:
 
 1. Set up GitHub account per [TTS GitHub guidelines](https://handbook.tts.gsa.gov/tools/github/#1-complete-your-profile)
 1. Join the [#admins-github](https://gsa-tts.slack.com/archives/C02KXM98G) Slack channel
-1. Tag `@github-admins-login` and request to add your GitHub username to the `18f/identity-core` group
+1. Tag `@github-admins-login` and request to add your GitHub username to the following teams/orgs:
+   1. [`18f/identity-core`][18f-identity-core]
+   2. [`GSA-TTS/identity-core`][gsa-tts-identity-core]
```
[the remark this answers is not in the document]
Comment (aduth) [reply _articles/github.md:147]: I think we need admins to help with permissions to the organization, but maintainers of the team (including you and me) can add individuals once they're in the organization.

[the remark this answers is not in the document]
Comment (zachmargolis) [reply _articles/github.md:147]: I'm hoping that the org admins have that super permission and can help us skip a step for first-time adds

[the remark this answers is not in the document]
Comment (zachmargolis) [reply _articles/github.md:147]: confirmed ✅

**Pull request merged**